src/components/Person/Person.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import classNames from 'classnames';
+import { PersonType } from '../../types';
+
+type Props = {
+  person: PersonType;
+  selectedPersonSlug: string;
+  isMother: PersonType | null;
+  isFather: PersonType | null;
+};
+
+export const Person: React.FC<Props> = ({
+  person,
+  selectedPersonSlug,
+  isMother,
+  isFather,
+}) => {
+  const {
+    name,
+    slug,
+    sex,
+    born,
+    died,
+    motherName,
+    fatherName,
+  } = person;
+
+  return (
+    <tr
+      key={slug}
```

Comment:
Remove the `key` prop from the `tr` element as it is not required in this context. The `key` prop is used when rendering a list of elements.